pandas-stubs/core/series.pyi

```diff
@@ -517,7 +517,11 @@ class Series(IndexOpsMixin[S1], ElementOpsMixin[S1], NDFrame):
     def __new__(
         cls,
         data: (
-            S1 | _DataLikeS1[S1] | dict[HashableT1, S1] | KeysView[S1] | ValuesView[S1]
+            S1
+            | _DataLikeS1[S1]  # ty: ignore[invalid-type-arguments]
```

Comment:
Unsure here why we need to subscribe with `S1` since it is already subscribed:
```python
_DataLikeS1: TypeAlias = (
    ArrayLike | dict[_str, np_ndarray] | Sequence[S1] | IndexOpsMixin[S1]
)
```